stdlib/xml/dom/expatbuilder.pyi

```diff
@@ -1,3 +1,103 @@
-from typing import Any
+# NOTE: Incomplete, lots of unannotated parameters currently
 
```

Comment:
This comment isn't helpful. It will either be clear to everyone looking at the stubs or outdated, depending on whether the stubs get improved later :)

Old versions of stubgen created comments like this, but they were removed at some point.

```suggestion
```